examples/tutorial.ipynb

```diff
@@ -297,20 +318,53 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "You can also get the sets of image files for each duplicated set using `imagelab.info`"
+    "You can also get exactly which images are there in each (exact/near) duplicated set using `imagelab.info`. The code below shows how to do that."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "imagelab.info['exact_duplicates']['sets']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`imagelab.get_index_to_path_mapping()` gets a mapping from internal image index to the path of the image. It is only applicable for datasets supplied using `data_path` or `filepaths` arguments in `Imagelab()`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "idx2path = imagelab.get_index_to_path_mapping()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
    "metadata": {
     "tags": []
    },
+   "source": [
+    "Using `imagelab.get_index_to_path_mapping()` and `imagelab.info['exact_duplicates']['sets']` we can get all sets of exact duplicate images in the dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
    "outputs": [],
    "source": [
-    "# This is a list of list, where each nested list is a set of exact duplicate images.\n",
-    "# Similarly, you can also retrieve near duplicate sets.\n",
-    "imagelab.info['exact_duplicates']['sets']"
+    "for i, duplicate_set in enumerate(imagelab.info['exact_duplicates']['sets']):\n",
+    "    print(f\"Set: {i}\")\n",
+    "    for idx in duplicate_set:\n",
+    "        print(idx2path[idx])"
```

Comment:
revisit this after potential source code index update, to spare users from such complex code

Reply:
removed